tests/metrics/estimate_gas.py

```diff
@@ -224,6 +225,11 @@ def transact(function, transaction):
         tx = function.transact(transaction)
         testerchain.wait_for_receipt(tx)
 
+    # First deposit ever is the most expensive, make it to remove unusual gas spending
```

Comment:
Why is the first deposit so expensive?

Reply:
Because first deposit initialize array of stakers so it costs more than usual